common/output.go

```diff
@@ -16,9 +16,14 @@ type outputMessageType uint8
 func (outputMessageType) Init() outputMessageType     { return outputMessageType(0) } // simple print, allowed to float up
 func (outputMessageType) Info() outputMessageType     { return outputMessageType(1) } // simple print, allowed to float up
 func (outputMessageType) Progress() outputMessageType { return outputMessageType(2) } // should be printed on the same line over and over again, not allowed to float up
-func (outputMessageType) Exit() outputMessageType     { return outputMessageType(3) } // exit after printing
-func (outputMessageType) Error() outputMessageType    { return outputMessageType(4) } // indicate fatal error, exit right after
-func (outputMessageType) Prompt() outputMessageType   { return outputMessageType(5) } // ask the user a question after erasing the progress
+
+// TODO: REVIEWERS: this next code was called Exit. But with followup jobs its not actually Exit.  Should we still call it exit
```

Comment:
I'd almost say that "exit" is something totally different now. Perhaps we should have EndOfJob and Exit, to indicate both states, as EndOfJob is now an ambiguous way to end something.

Reply:
Yeah, I'm not entirely happy with what I've done here.  

@Zeza-msft you know a lot about how the JSON output is used, which is where my change is most likely to be a breaking one.  What do you think?

Reply:
@JohnRusk EndOfJob is fine, as Stg Exp will rework the JSON parsing anyway due to Prompt messages.

Reply:
Adele, I realized that anything listening to our stdout will have to be monitoring for unexpected process exit anyway, so it must be already on the lookout for closure (EOF) of our stdout.  So for now, we don't need a separate exit signal. But I've left a TODO in the code to consider whether we might need one in the future.

Reply:
Good point. Yeah, this should probably just be left as a future work item to make output more clear in the case that we do ever start adding cleanup behavior to azcopy.